src/bitcoinrpcclient/bitcoinrpcclient.go

```diff
@@ -3,37 +3,48 @@ package bitcoinrpcclient
 import (
 	"encoding/json"
 	"fmt"
-	"log"
+	"net/url"
+	"os"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/btcsuite/btcutil"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
This change slipped in a bit. I can pull it out and submit a separate PR for a more consistent use of logrus.